cvd/allocd/allocd.cpp

```diff
@@ -34,9 +34,9 @@
 #include <thread>
 
 #include "common/libs/fs/shared_fd.h"
-#include "host/libs/allocd/alloc_utils.h"
-#include "host/libs/allocd/request.h"
-#include "host/libs/allocd/resource_manager.h"
+#include "alloc_utils.h"
+#include "request.h"
+#include "resource_manager.h"
```

Comment:
nit: should these now be under `allocd/`?

Should also be reordered.

Reply:
reorders done